pygmt/src/plot.py

```diff
@@ -87,6 +89,8 @@ def plot(self, data=None, x=None, y=None, size=None, direction=None, **kwargs):
     size : 1-D array
         The size of the data points in units specified using ``style``.
```

Comment:
Hm. But if `size` is used with `symbol` the unit for `size` is given by the GMT / PyGMT defaults, correct?
Using `style` for specifing the unit in this case would be super confusing, e.g.:
```python
...
size=[0.5, 0.8, 0.9],
symbol=["s", "c", "d"],
style="c",  # now centimeters, not circle
...
```

Reply:
Actually, it produces three circles with different sizes.